rc-events-service/src/main/scala/com/epam/random_coffee/events/services/impl/EventServiceImpl.scala

```diff
@@ -0,0 +1,42 @@
+package com.epam.random_coffee.events.services.impl
+
+import com.epam.random_coffee.events.model.{ Event, EventId }
+import com.epam.random_coffee.events.repo.EventRepository
+import com.epam.random_coffee.events.services.EventService
+
+import java.util.UUID
+import scala.concurrent.{ ExecutionContext, Future }
+
+class EventServiceImpl(repo: EventRepository)(implicit ec: ExecutionContext) extends EventService with Doobie_support {
+
+  override def create(eventName: String): Future[Event] = {
+    val id = EventId(UUID.randomUUID().toString)
+    val event = Event(id, eventName)
+    repo.save(event)
+    Future(event)
```

Comment:
1) As I said before, if you want to create a Future and the result is already known, use `Future.successfull` instead
2) This code contains a bug. Write a test that imitates a case when saving to database has failed, and then check if this method fails as well

Reply:
Fixed.
Added a test for unsuccessful saving to database